### DISCUSSION TITLE
How to set the template for this case

### DISCUSSION THREAD

**Discussion by darmau** [umputun/remark42#1620] [Q&A]
I have a push server and I just need to append content to the end of the url and I can receive the corresponding information. Like this: `https://push.test.com/{token}/{content}`, I will receive the content message.

Can remark42 implement this kind of webhook push? How to write the template?

**Comment by paskal** [comment]
The content length could easily be longer than the URL limit, which is [around 2000 symbols](https://stackoverflow.com/a/417184/961092). We should not implement this particular type of webhook. Do you have any problems with implementing a client for [existing webhook notifications](https://remark42.com/docs/configuration/notifications/)?
